### PR TITLE
Adding mwupload queue into rabbitmq

### PR DESCRIPTION
**Pull request by maxixam**
change on peatio side.  Need to update docker file to start the daemon in workbench from a different PR.

Publishing an event is still not fully functioning, debugging now.

### CODE REVIEW

File: app/workers/worker/deposit_mwupload.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+        rescue Exception => e
+          begin
+            Rails.logger.error { "Failed to process upload. See exception details below." }
+            report_exception(e)
+        end
+      end
+    end
+  end
+
```
Comment (jordansaints) [inline app/workers/worker/deposit_mwupload.rb]: are you missing an `end` here? indentation and/or count looks off

File: lib/wallet_client/geth.rb [hunk 2]
```diff
@@ -142,6 +142,7 @@ def json_rpc(method, params = [])
           'Content-Type' => 'application/json' }
       response.assert_success!
       response = JSON.parse(response.body)
+      #Rails.logger.debug { "method = #{method}, Params = #{params}, Response = #{response}" }
```
Comment (shiqicao) [inline lib/wallet_client/geth.rb]: remove

File: app/workers/worker/deposit_mwupload.rb (hunk 1 above)
Comment (maxixam) [reply app/workers/worker/deposit_mwupload.rb]: we counted and it should be right.  changed the indent but not reflected correctly here.

File: lib/wallet_client/geth.rb (hunk 2 above)
Comment (maxixam) [reply lib/wallet_client/geth.rb]: will do.

**Pull request merged**